test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/project.json

```diff
@@ -25,7 +25,7 @@
       "dependencies": {
         "moq.netcore": "4.4.0-beta8",
         "dotnet-test-xunit": "1.0.0-rc2-173361-36",
-        "System.Diagnostics.TraceSource": "4.0.0-rc3-24210-10",
+        "System.Diagnostics.TraceSource": "4.0.0",
         "Microsoft.NETCore.App": {
           "type": "platform",
           "version": "1.0.0-rc3-004338"

```

Comment:
Because we restore Microsoft.NETcore.App from CLI shares folder, not from myget server, I don't why, But anyway, I checked in this first to make a new production build and update test later